docs/Developers/Review.rst

```diff
@@ -65,6 +65,14 @@ The review process for coala is as follows:
    ``corobo mark wip <URL>`` or ``corobo mark pending <URL>``.
 3. The developers will acknowledge the commits by writing
 
+   * In case a member is reviewing it:
```

Comment:
hmm, maybe "newcomer" is better than member

Reply:
There are only two distinctions as per coala : member or maintainer. I don't think any change is required. @adtac